src/Lock.ts

```diff
@@ -19,6 +19,7 @@ export abstract class Lock {
   protected _refreshInterval?: ReturnType<typeof setInterval>
   protected _refreshing = false
   protected _acquired = false
+  protected _acquiredExternally = false
```

Comment:
documentation for the new param is missing

Reply:
updated README